tf2_eigen/CMakeLists.txt

```diff
@@ -26,11 +26,12 @@ if(BUILD_TESTING)
   ament_add_gtest(tf2_eigen-test test/tf2_eigen-test.cpp)
   if(TARGET tf2_eigen-test)
     target_include_directories(tf2_eigen-test PUBLIC
-      include
-      ${tf2_INCLUDE_DIRS}
-      ${tf2_ros_INCLUDE_DIRS}
-      ${Eigen_INCLUDE_DIR}
-    )
+      include)
+    ament_target_dependencies(tf2_eigen-test
+      "tf2"
+      "tf2_ros")
+    target_include_directories(tf2_eigen-test PUBLIC
+      ${Eigen_INCLUDE_DIR})
```

Comment:
How about using `EIGEN3_INCLUDE_DIR` here (as well as in line 19) and removing the custom variable `Eigen_INCLUDE_DIR`?

Reply:
Sorry, I skipped the last part of the other comment. Now is fine.